src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRent = 40;
+  const smallDiscount = 20;
+  const bigDiscount = 50;
+
+  return days < 3
+    ? days * dailyRent
+    : days < 7
+      ? days * dailyRent - smallDiscount
+      : days * dailyRent - bigDiscount;
```

Comment:
It's awesome that you've found out how to use ternary operator, but nesting here just decrease code readability. Let's rewrite it using if / else statement